gofsutil_fs.go

```diff
@@ -169,24 +170,24 @@ func (fs *FS) MultipathCommand(ctx context.Context, timeoutSeconds time.Duration
 
 // Info linux returns (available bytes, byte capacity, byte usage, total inodes, inodes free, inode usage, error)
 // for the filesystem that path resides upon.
-func FsInfo(path string) (int64, int64, int64, int64, int64, int64, error) {
+func (fs *FS) fsInfo(ctx context.Context, path string) (uint64, uint64, uint64, uint64, uint64, uint64, error) {
```

Comment:
this method is only called by driver to make a  csi volumeUsage object which has int64 types . so why return uint64 and force type conversion ?

Reply:
@francis-nijay thank you for making this PR. I would second Jai's question here. Since we are using this function to replace the k8s Info() method, everybody here who is using this to replace Info would need to have their code set up to use int64 to make things work. If we want to go back to int64, I grabbed this branch locally last night and changed everything to int64 and verified that it works in my setup -- I could just commit and push those changes up if that sounds good to you.

Reply:
@jooseppi-luna I ve changed the returned type back to int64 post discussion over email. Good to be merged.